spec.bs

```diff
@@ -1523,3 +1523,50 @@ Information of the auction's leading bid so far when ranking scored bids.
 :: Null or a [=generated bid=]. The leading bid of the auction so far.
 
 </dl>
+
+# Privacy Considerations # {#privacy-considerations}
+
+FLEDGE aims to advance the privacy of remarketing and custom audience advertising on the web, so
+naturally privacy considerations are paramount to FLEDGE’s design.  Partitioning data by site is
+the central mechanism to prevent joining a user’s identity across sites:
+- Interest group definitions come from just one site, the site that called
+  {{Navigator/joinAdInterestGroup()}}.
+- Bidding (`generateBid()`) and scoring (`scoreAd()`) scripts are exposed to interest group data
+  from the joining site, and data from the site that will display the ad, but these scripts are run
+  in strict isolation to prevent leaking this cross-site information.
+- The browser can enforce that trusted bidding and scoring signals are fetched from servers
+  complying with certain privacy requirements, e.g. not performing event-level logging.
+- Ads are rendered in [Fenced Frames](https://wicg.github.io/fenced-frame/) that isolate them from

```

Comment:
nit: I'd make this all lowercase since this is basically referring to the element name. More specifically, you can make a PR to export this definition https://github.com/WICG/fenced-frame/blob/master/spec.bs#L186 and use `<{fencedframe}>` if you'd like.

Reply:
I sent in a PR at https://github.com/WICG/fenced-frame/pull/75 - seems useful to export.